test/VirtualDeviceInvoker.test.js

```diff
@@ -183,41 +185,6 @@ describe("virtual device runner", () => {
         });
     });
 
-    describe("skip and only tests", () => {
-        beforeAll(() => {
```

Comment:
this tests are unrelated to the changes, why remove them?

Reply:
These tests are extraneous - they are a cut and paste from the VirtualAlexaInvoker - we don't need the same tests in two places.